src/aiida/orm/utils/mixins.py

```diff
@@ -181,22 +183,21 @@ class Sealable:
 
     SEALED_KEY = 'sealed'
 
-    __qb_fields__ = [
-        add_field(
-            SEALED_KEY,
-            dtype=bool,
-            doc='Whether the node is sealed',
-        ),
-    ]
+    class Model(pydantic.BaseModel):
+        sealed: bool = MetadataField(description='Whether the node is sealed')
```

Comment:
Is the `SEALED_KEY` still in use?

Reply:
Yes,  it is now used in the new `sealed` property